.env.example

```diff
@@ -1,2 +1,4 @@
 APP_PORT=3000
+RG_SSO_CLIENT_ID=
```

Comment:
Can call this `RSO_CLIENT_ID`, the terminology we're going for is calling these oauth clients "rso clients"

Reply:
done!